src/main.py

```diff
@@ -76,8 +77,18 @@ def create_application() -> FastAPI:
 
     # Include routers
     app.include_router(health.router)
-
-    #TODO: Add MCP Server Integration
+    app.include_router(delta.router)
```

Comment:
Is this new? I don't see any new router in this PR

Reply:
I accidentally committed it in the previous PR - thought I already merged reviewed commit. 
https://github.com/kbase/cdm-mcp-server/commit/26a0cf36de15e7cde04f20e6d01ff8ac